qiskit/transpiler/passmanager.py

```diff
@@ -212,7 +212,9 @@ def callback_func(**kwargs):
         Returns:
             The transformed circuit(s).
         """
-        if isinstance(circuits, QuantumCircuit):
+        if not self._pass_sets and output_name is None and callback is None:
+            return circuits
+        elif isinstance(circuits, QuantumCircuit):
```

Comment:
This technically doesn't need to be `elif` as the return in the if you're adding implies this is else.

Reply:
```suggestion
        if isinstance(circuits, QuantumCircuit):
```